code/go/0chain.net/core/transaction/http.go

```diff
@@ -105,10 +105,15 @@ func makeSCRestAPICall(scAddress string, relativePath string, params map[string]
 		"Access-Control-Allow-Origin": "*",
 	}
 
+	options := []resty.Option{
+		resty.WithTransport(transport),
```

Comment:
If the transport is the same as the default transport that is used in restry.New(), then we don't have to use it here again.

Reply:
fixed